.azure/pipelines/helix-matrix.yml

```diff
@@ -1,6 +1,9 @@
 # We only want to run full helix matrix on master
-pr: none
-trigger: none
+pr:
```

Comment:
remove before merge

Reply:
Curious: What's the usefulness of this?

Reply:
We don't run all the helix queues on PRs, this hack makes them run for the PRs so if you are investigating issues on helix matrix or other non PR queues, you can do this trick as opposed to manually kicking off builds in the yaml... which is an option, but I find this easier... just need to make sure you don't merge the yml changes :)  Which is why I always put in a comment to make sure I don't forget